pkg/spireapi/types_test.go

```diff
@@ -119,10 +119,8 @@ func TestHTTPSSPIFFEProfileEquality(t *testing.T) {
 	idA := HTTPSSPIFFEProfile{EndpointSPIFFEID: spiffeid.RequireFromString("spiffe://a/endpoint")}
 	idB := HTTPSSPIFFEProfile{EndpointSPIFFEID: spiffeid.RequireFromString("spiffe://b/endpoint")}
 
-	assert.True(t, idA.Equal(idA))
```

Comment:
We should still be asserting that two profiles that are identical are "equal". To silence the lint, we can create another profile that matches idA or idB for the comparison.